src/vocgui/templates/public_upload.html

```diff
@@ -79,11 +79,23 @@
           });
         });
       });
+
+      function submit_handler() {
+        alert("Vielen Dank für Ihren Beitrag!");
```

Comment:
I personally would render this message in HTML after successful upload.

Reply:
I agree